config/application.rb

```diff
@@ -90,6 +89,9 @@ class Application < Rails::Application
       'SCIENCE' => 'Science Library (Li and Ma)',
       'SPEC-COLL' => 'Special Collections',
       'SPEC-DESK' => 'Special Collections',
+      # TODO: remove this first entry BORROW_DIRECT after FOLIO. SUL-BORROW-DIRECT is the new FOLIO code.
```

Comment:
```suggestion
      # TODO: remove this first entry BORROW_DIRECT after migrating off of Symphony. SUL-BORROW-DIRECT is the new FOLIO code.
```

Reply:
I will add this. The ILL code too will be replaced but can leave off that comment until we are addressing that particular code.